### ISSUE TITLE
Suggestion: bookmarks: wider click area

### ISSUE THREAD

**Issue by avih** [sumatrapdfreader/sumatrapdf#1920]
This apply equally to both bookmarks and favorites.

Currently the hover-highlight and clickable area of a bookmark is its text width.

If a bookmark title is short (few characters) then it's harder to aim the mouse. It would be nicer if the active area would extend to the rightmost of the bookmarks pane, not unlike other menus where the whole width is active even with short text.

Optionally, similarly extend the active area also to the left. This could apply either to the text, or to the collapse-icon, where applicable.

**Comment by GitHubRulesOK** [comment]
@kjk will need to confirm, however I think this is a "feature" of the tree control as supplied by Microsoft Visual Studio, so whilst there may possibly be a "FULLROWSELECT" option such as to include white space on the right, I do not think it can be applied independently to the left?

**Comment by avih** [comment]
> Thus the tree is either the short lengths with vertical line indicators (as at present) OR full width with no parent child indicators (as seen in windows explorer trees)

Right. I wasn't sure who implements the tree view.
However, now that you mention it, windows explorer is the closest to my suggestion, I think:
- The full row is highlighted and clickable, for both "leaves" and items with children.
- For itms with children: single click selects the item, and double click toggles the children view.

Also, one could assume that windows users are familiar with Windows explorer UI paradigm.

Maybe add an option to choose between explorer-like tree and the current tree?

**Comment by avih** [comment]
ping?

**Comment by GitHubRulesOK** [comment]
ping-pong? I appreciate the wider bars but how hard would it be to add a toggle ? 
![image](https://user-images.githubusercontent.com/12754913/116606403-9778e280-a928-11eb-824a-1141d3b38f92.png)

**Comment by avih** [comment]
Thanks. I tried it for a short while and subjectively it's quite nicer than before, despite the lack of vertical lines.

Looking at the commit, however, it's not obvious to me whether it's supposed to be configurable or not.

Is this configurable? if yes, how?

(I tried to add `fullRowSelect = false` at `SumatraPDF-settings.txt`, but it seems it was rejected and moved to the bottom of the file as an unrecognized option)
